src/control/cmd/drpc_test/hello/drpc_test.pb.go

```diff
@@ -112,7 +112,7 @@ func (Function) EnumDescriptor() ([]byte, []int) {
 	return file_drpc_test_proto_rawDescGZIP(), []int{1}
 }
 
-//*
+// *
```

Comment:
Should probably re-gen this pb file instead of manually editing. Looks like the more recently regenerated files weren't hit by anything like this.

Reply:
The source .proto files had comments that resulted in this weird stuff that go fmt wanted to fix. Just changed the comment style to match the other sources.